Call for the Conan Center 1,000 recipes event (#202)
conan-io/conan-io.github.io: _posts/2021-09-22-1000-recipes-call.markdown

---
layout: post
comments: false
title: "Achievement Unlocked: Conan Center Hits 1,000 Recipes (and Counting)"
meta_title: "Achievement Unlocked: Conan Center Hits 1,000 Recipes (and Counting)"
description: "Call for the Conan Center 1,000 recipes celebration event"
meta_description: "Call for the Conan Center 1,000 recipes celebration event"
---

<p class="centered">
    <img src="{{ site.baseurl }}/assets/post_images/2021-09-22/1000_recipes.jpg" align="center" alt="Thank you!"/>
</p>

On August 27, 2019, the conan-center-index repository was created, and Lasote
[uploaded the first recipe](https://github.com/conan-io/conan-center-index/commit/b39b3889147fd84643e1b65a433934343eb70ff4):
zlib.

> Almost exactly two years later, SpaceIM (who else)

Reviewer: ```suggestion
Almost exactly two years later, SpaceIM (who else)
```
Maybe I would change this `(who else)` to something like `(one of the most important Conan Center contributors alongside other like ... and name some of the contributors)`

Reviewer 2: Agreed. Companies that read the blog may not be familiar with the CCI, nor with the contributors

Author: This is just a small "wink" to the community, it's not a big deal 🙂

Reviewer 3: Agree this might be a bit confusing for people that are not that into conan-center-index. I'd add the change @czoido suggests or maybe just omit it

Author: ```suggestion
Almost exactly two years later, SpaceIM
```

> [uploaded libsamplerate](https://github.com/tapia/conan-center-index/commit/9bc281eac017bd33a5cf221e289a0a10ad0ffdfb),

Reviewer 2: ```suggestion
[uploaded libsamplerate](https://github.com/conan-io/conan-center-index/commit/9bc281eac017bd33a5cf221e289a0a10ad0ffdfb),
```

Author: Ouch... good catch!!

and Conan Center officially reached **the astounding number of 1.000 recipes**.

Since its inception, Conan Center has grown a lot. Almost 400 contributors, 10,000 commits, 5,500
pull requests, and, most of all, an incredible community helping us every day to make it even bigger
and better.

Of course, that’s something we want to celebrate. And we’d like you to join us!

> So on Thursday, September 30 we’re going to host an online event for all Conan Center contributors

Reviewer 2: ```suggestion
So on Thursday, September 30, at 17:00 (UTC) we’re going to host an online event for all Conan Center contributors
```

and users. We will talk about how big Conan Center has become, and what can we expect from it in the
future.

### Event Activities

* Welcome and Introduction
* Short Presentation - Conan Center: Much More Than Recipes

> * Big raffle for attending contributors: Nintendo Switch or Oculus VR headset

Author: ```suggestion
* Big raffle: if you're a Conan Center contributor you can win a Nintendo Switch!
```

* Panel with community contributors and reviewers - Full Attendee Participation

The event will last about an hour.

### Event Details:

Date: September 30, 2021

> Time: 17:00 UTC / 13 EST

Author: ```suggestion
Time: 17:00 UTC / 13:00 EST
```

Duration: about 60-90 minutes

Event Link: [https://www.youtube.com/watch?v=PjiJ-3sxKbs](https://www.youtube.com/watch?v=PjiJ-3sxKbs)

### CODE OF CONDUCT

> To make sure this event is a safe space for everyone, all attendees, speakers,and organizers are

Reviewer 2: ```suggestion
To make sure this event is a safe space for everyone, all attendees, speakers, and organizers are
```
